internal/infrastructure/db/repositories/device_definition_repo.go

```diff
@@ -180,3 +181,24 @@ func (r *deviceDefinitionRepository) GetOrCreate(ctx context.Context, source str
 	}
 	return dd, nil
 }
+
+func (r *deviceDefinitionRepository) FetchDeviceCompatibility(ctx context.Context, makeID, integrationID, region string) (models.DeviceDefinitionSlice, error) {
+	res, err := models.DeviceDefinitions(
+		qm.InnerJoin("device_integrations ON device_definitions.id = device_integrations.device_definition_id"),
```

Comment:
There are some examples of doing this with the constants [here](https://github.com/volatiletech/sqlboiler#query-mod-system). Maybe it's not worth it?